BUILD.bazel

```diff
@@ -7,9 +7,9 @@ load(
 
 _JAVA_COPTS = [
     "-source",
-    "1.8",
+    "23.0.2",
```

Comment:
I don't think `-source` and `-target` support specifying minor versions, can you try to use `17`?

Reply:
I tried `17` and `23`, and the `bazelisk  run :google_java_format --enable_workspace` command still failed.